src/components/Pursuance/PursuanceMenu.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Navbar, NavItem, Nav, ButtonGroup, Button, DropdownButton, MenuItem } from 'react-bootstrap';
+import { NavLink } from 'react-router-dom';
+import PursuanceMenuItem from './PursuanceMenuItem';
+import FaCheckSquareO from 'react-icons/lib/fa/check-square-o';
+import CommentsO from 'react-icons/lib/fa/comments-o';
+import PaperClip from 'react-icons/lib/fa/paperclip';
+import Group from 'react-icons/lib/fa/group';
+import Rocket from 'react-icons/lib/fa/rocket';
+import PlusCircle from 'react-icons/lib/fa/plus-circle';
+
+const PursuanceMenu = ({ pursuanceId }) => {
+  return (
+    <ButtonGroup vertical>
+      <Button>
+        <PursuanceMenuItem
+          pursuanceId={pursuanceId}
+          label='New'
+          action='add'
```

Comment:
@Spetastian Is calling this `new` safe or do you think this might interfere with a keyword somewhere?

I'm picturing this as being a sort of drop down that lets you create a new task or a new pursuance, FWIW.

Reply:
Yes, i do think its safe.

Regarding the behavior of each item in the menu. If it should lead to a new route or just display a dropdown or modal etc.. Lets implement that case by case. Right now most of these items are just dummies. When we get to the point where we should implement the `add` functionality here we might remove the MenuItem component for that and just show a dropdown instead.